pages/admin/dashboard/modules.tsx

```diff
@@ -38,13 +55,13 @@ const Modules: Page = () => {
             </Flex>
             <SimpleGrid minChildWidth="20rem" spacing={10}>
                 {modules.map(
-                    ({ moduleId, title, tool, lastUpdated, author }) => (
+                    ({ moduleId, title, toolID, lastUpdated, createdBy }) => (
                         <ModuleCard
                             key={moduleId}
                             title={title}
-                            tool={tool}
+                            tool={toolID}
                             lastUpdated={lastUpdated}
-                            author={author}
+                            author={createdBy}
```

Comment:
Good catch- `createdBy` is an array atm. Since we are only looking at "createdBy" that should only be ONE person- I think we should make `createdBy` a string- any thoughts about making it a string instead of an array??

Reply:
I think we can leave `author` prop as a string and pass in `createdBy.join(', ')` to format it as a string